Doc/c-api/type.rst

```diff
@@ -492,11 +492,15 @@ The following functions and structs are used to create
          See :ref:`PyMemberDef documentation <pymemberdef-offsets>`
          for details.
 
-      The following fields cannot be set at all when creating a heap type:
+      .. versionchanged:: 3.14
 
-      * :c:member:`~PyTypeObject.tp_vectorcall`
-        (use :c:member:`~PyTypeObject.tp_new` and/or
-        :c:member:`~PyTypeObject.tp_init`)
+         The field :c:member:`~PyTypeObject.tp_vectorcall` can only be set
+         since Python 3.14. On older versions, use
+         :c:member:`~PyTypeObject.tp_new` and/or
+         :c:member:`~PyTypeObject.tp_init`.
```

Comment:
FWIW, you should use tp_new and/or tp_init on *all* versions; in 3.14 you can optimize after you have them.
(I removed this in my PR.)